kolibri/utils/conf.py

```diff
@@ -55,3 +55,21 @@ def __initialize_options():
 
 
 OPTIONS = SimpleLazyObject(__initialize_options)
+
+# Generate an options.ini inside the KOLIBRI_HOME as default config
+try:
+    options_path = os.path.join(KOLIBRI_HOME, "options.ini")
+    if not os.path.exists(options_path):
+        file = open(options_path, "w")
+        for k, v in OPTIONS.items():
+            file.write("# [%s] \n" % (k))
+            loop_count = 0
+            for k_child, v_child in v.items():
+                loop_count += 1
+                if loop_count != len(v):
+                    file.write("# %s = %s \n" % (k_child, v_child))
+                else:
+                    file.write("# %s = %s \n\n" % (k_child, v_child))
+        file.close()
```

Comment:
Let's wrap this code in a function: `generate_empy_options_file` and put it in `options.py`

For the actual check to see if the options file exists, we can add that as a function inside `sanity_checks.py` and invoke it during CLI start at the same time as other sanity checks.

Reply:
regarding the place to put the sanity check for start command, I think it would be good to put it in the beginning of the command: https://github.com/learningequality/kolibri/blob/release-v0.13.x/kolibri/utils/cli.py#L421